packages/web3-core-method/types/index.d.ts

```diff
@@ -17,6 +17,8 @@
  * @date 2018
  */
 
+import {JsonRpcPayload} from 'web3-core-helpers';
```

Comment:
This import isn't in usage and can get removed because it is totally obvious

Reply:
@nivida - Tbh I've just scanned your review - each comment provides a rationale for the change. As I note in the opening comment, I think many of the changes *are* legitimate. But they're also out of scope, add lots of complexity, reduce the clarity of the changes and can be introduced separately. 40 files can be removed from #3190 (non of them test files) with all tests passing. It's not a minimized set.

Reply:
I see those points as well but instead of just reverting those smaller improvements should we probably also create issues in the same time to track those un-clean parts of the ``web3.js`` code base. This is probably also a good chance to create issues labeled with ``good first issue``. WDYT? :)

Reply:
I've created issue #3334, #3335, and #3336 to not forget about those required types improvements and to show what I thought with the comment above.